README.md

```diff
@@ -110,6 +110,40 @@ Support for Go versions on the OpenTelemetry Collector is updated as follows:
 
 Official OpenTelemetry Collector distro binaries may be built with any supported Go version.
 
+## Verifying the images signatures
+
+> **Note**: To verify a signed artifact or blob, first [install Cosign](https://docs.sigstore.dev/system_config/installation/), then follow the instructions below.
+
+We are signing the images `otel/opentelemetry-collector` and `otel/opentelemetry-collector-contrib` using [sigstore cosign](https://github.com/sigstore/cosign) tool and to verify the signatures you can run the following command:
+
+```console
+$ cosign verify \
+  --certificate-identity=https://github.com/open-telemetry/opentelemetry-collector-releases/.github/workflows/base-release.yaml@refs/tags/<RELEASE_TAG> \
+  --certificate-oidc-issuer=https://token.actions.githubusercontent.com \
+  <OTEL_COLLECTOR_IMAGE>
+```
+
+where:
+
+- `<RELEASE_TAG>`: is the release that you want to validate
+- `<OTEL_COLLECTOR_IMAGE>`: is the image that you want to check
+
+Example:
+
+```console
```

Comment:
Can we put this into a [`<details>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details) block? It's pretty verbose

Reply:
Actually, I don't think the details block helps much here, it's just that the lines are too long. I think we can merge without changing this